jwst/regtest/test_miri_lrs_slitless.py

```diff
@@ -104,14 +96,14 @@ def test_miri_lrs_slitless_tso1(run_tso1_pipeline, fitsdiff_default_kwargs, step
 
 @pytest.mark.bigdata
 @pytest.mark.parametrize("step_suffix", ["flat_field", "srctype", "calints", "x1dints"])
-def test_miri_lrs_slitless_spec2(run_spec2_pipeline, rtdata_module, fitsdiff_default_kwargs,
+def test_miri_lrs_slitless_tso_spec2(run_tso_spec2_pipeline, rtdata_module, fitsdiff_default_kwargs,
     step_suffix):
     """Compare the output of a MIRI LRS slitless calwebb_tso-spec2 step."""
```

Comment:
Minor nit, but I'd change "calwebb_tso-spec2 **step**" to "calwebb_tso-spec2" **pipeline**.